Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalIsoTrkFromAlCaRaw_cff.py

```diff
@@ -0,0 +1,26 @@
+import FWCore.ParameterSet.Config as cms
+
+#--------------------------------------------------------------------
+# AlCaReco filtering for running on dedicated AlCaRaw: AlCaHcalIsoTrk
+#--------------------------------------------------------------------
+
+from Calibration.HcalAlCaRecoProducers.alcaHcalIsotrkProducer_cff import alcaHcalIsotrkProducer as _alcaHcalIsotrkProducer
+from Calibration.HcalAlCaRecoProducers.alcaHcalIsotrkFilter_cfi import alcaHcalIsotrkFilter as _alcaHcalIsotrkFilter
+
+
+alcaHcalIsotrkFromAlCaRawProducer = _alcaHcalIsotrkProducer.clone(
+    labelTrack = "hltMergedTracksSelector",
+    labelVertex = "hltTrimmedPixelVertices",
+    labelEBRecHit = cms.InputTag("hltEcalRecHit", "EcalRecHitsEB", ""),
+    labelEERecHit = cms.InputTag("hltEcalRecHit", "EcalRecHitsEE", ""),
```

Comment:
```suggestion
    labelEBRecHit = ("hltEcalRecHit", "EcalRecHitsEB", ""),
    labelEERecHit = ("hltEcalRecHit", "EcalRecHitsEE", ""),
```

you should be able to drop cms type specifications.